src/nvim/terminal.c

```diff
@@ -161,6 +161,8 @@ static Map(int, int) *color_indexes;
 static int default_vt_fg, default_vt_bg;
 static VTermColor default_vt_bg_rgb;
 
+static void buf_set_term_title(buf_T *buf, String title);
```

Comment:
With the following block the static function declarations are included:
```c
#ifdef INCLUDE_GENERATED_DECLARATIONS
# include "terminal.c.generated.h"
#endif
```
Static functions should be defined after this include.
Generated files can be found in `./build/src/nvim/auto/...`. Don't know of any documentation other than the implementing pull request.